src/ui/Roster.ts

```diff
@@ -80,6 +84,78 @@ export default class Roster {
       }
    }
 
+   private initCollapisible()
+   {
+      let collapsibleElement = $('.jsxc-collapsible');
+
+      for(let i = 0; i < collapsibleElement.length; i++)
+      {
+         $(collapsibleElement[i]).off('click').on('click', function()
+         {
+            this.classList.toggle('jsxc-active');
+            let content = $(this).next();
+            if (content.first().css('display')!=='none'){
+                content.first().css('display','none');
+                content.first().css('max-height','');
+            } else {
+                content.first().css('display','');
+                content.first().css('max-height','100%');
+            }
+         });
+
+         $(collapsibleElement[i]).next().css('flex-grow', 'unset');
+      }
+
+      if (this.contactList[0].childNodes.length>0)
+      {
+          $(this.contactList[0]).css('display','');
+          $(this.contactList[0]).css('max-height','100%');
+      }
+   }
+
+   private addContactFilter()
+   {
+      $('#filterlist').off('keyup').on('keyup',function(e)
```

Comment:
JSXC is used in many environments, so we should prefix this with `jsxc-`.

Reply:
added jsxc- prefix